### PR TITLE
ressting widths

### PR DESCRIPTION
**Pull request by NachmanBerkowitz**
When resetting the table the widths will now also reset.

### CODE REVIEW

File: projects/table-builder/src/lib/components/column-builder/column-builder.component.ts [hunk 1]
```diff
@@ -75,6 +79,16 @@ export class ColumnBuilderComponent implements OnInit {
     }
   }
 
+  mapWidth = (currentAndPrevious : [number, number]) => {
+    const previousWidth = currentAndPrevious[0];
```
Comment (e-davidson) [inline projects/table-builder/src/lib/components/column-builder/column-builder.component.ts:83]: destructure

File: projects/table-builder/src/lib/components/column-builder/column-builder.component.ts [hunk 2]
```diff
@@ -75,6 +79,16 @@ export class ColumnBuilderComponent implements OnInit {
...
+    const currentWidth = currentAndPrevious[1]
+    if(currentWidth >= 0){
```
Comment (e-davidson) [inline projects/table-builder/src/lib/components/column-builder/column-builder.component.ts:85]: use a method name or variable names that describe the why of why we're using this logic.

**Comment by NachmanBerkowitz** [comment]
> destructure 

Is this better?

**Comment by NachmanBerkowitz** [comment]
> destructure 

And now?

**Review by e-davidson — APPROVED** [review]

**Pull request merged**